### PR TITLE
Fixed according to feedback on #35

### PR DESCRIPTION
**Pull request by grainier**
Did several changes according to the feedback received on #35. Please review and merge.

### CODE REVIEW

File: components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/util/MetricsConstants.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.metrics.impl.util;
+
+public class MetricsConstants {
```
Comment (chrishantha) [inline components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/util/MetricsConstants.java]: I don't like having a separate class for constants in Metrics. All these constants are used in MetricServiceImpl. Shall we move all these declarations to the MetricServiceImpl?

File: components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/MetricServiceImpl.java [hunk 2]
```diff
@@ -86,25 +87,15 @@
...
-    private static final String SYSTEM_PROPERTY_METRICS_ENABLED = "metrics.enabled";
-    private static final String SYSTEM_PROPERTY_METRICS_ROOT_LEVEL = "metrics.rootLevel";
-
-    /**
-     * Name of the root metric. This is set to empty string.
-     */
-    private static final String ROOT_METRIC_NAME = "";
-
-    /**
-     * Hierarchy delimiter in Metric name
```
Comment (chrishantha) [inline components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/MetricServiceImpl.java:98]: Please keep these comments

File: components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/util/MetricsConstants.java (hunk 1 above)
Comment (grainier) [inline components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/util/MetricsConstants.java]: @chrishantha Constants moved to `MetricServiceImpl` on [commit:0e28587](https://github.com/grainier/carbon-metrics/commit/0e285876fde290b95b0541c5432be7edf7e9be05)

File: components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/MetricServiceImpl.java (hunk 2 above)
Comment (grainier) [inline components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/MetricServiceImpl.java:98]: @chrishantha Reverted on [commit:0e28587](https://github.com/grainier/carbon-metrics/commit/0e285876fde290b95b0541c5432be7edf7e9be05)

**Pull request merged**